core/random.rb

```diff
@@ -182,3 +182,5 @@ def bytes(length)
     s
   end
 end
+
+Random::DEFAULT = Random.new
```

Comment:
I'd suggest adding this line to the end of core/zed.rb. The Rubinius core libraries load in a sort of phased process where some functionality isn't ready when a particular file is loaded. The core/zed.rb file is loaded last and all basic Ruby functionality should be available at that time, but some pieces of the core library still need to be assembled.